### PR TITLE
Desktop/mobile apps need branding to work with OIDC

### PR DESCRIPTION
**Pull request by ib-mlatin**
Make a note that the desktop/mobile apps do not work by default with Azure AD.

### CODE REVIEW

**Comment by mmattel** [comment]
@ib-mlatin many thanks for your PR. I need to rewrite it a bit but this is not a big deal.

**Comment by mmattel** [comment]
Desktop/mobile apps do not work by default with OICD (MS-Azure is just one possible Idp...)
Improved the text and relocated to the parent level (OIDC)

**Review by phil-davis — APPROVED** [review]
English looks OK to me.

**Review by michaelstingl — CHANGES_REQUESTED** [review]
Comment for general OIDC doesn't make sense. Needs only comment on `ms-azure-setup.adoc` page.

File: modules/admin_manual/pages/configuration/user/oidc/oidc.adoc [hunk 1]
```diff
@@ -104,6 +104,8 @@ xref:configuration/user/oidc/ms-azure-setup.adoc[Example Setup Using Microsoft A
 
 ownCloud desktop and mobile clients detect whether OIDC is available (service discovery) and use this login method when a new account is created.
 
+NOTE: The desktop and mobile apps (clients) have a default client ID and secret hardcoded, used for ownCloud's oauth2 app. When using Kopano as IDP, it does not predefine a client ID and secret. You can use the default ones from the client to configure Kopano properly. Some IDP's like MS-Azure, these and other required parameters come from the IPD and must be coded into the client. Note that each IDP has different requirements. Get in touch with ownCloud for a branding subscription to customize the clients according your needs.
```
Comment (phil-davis) [inline modules/admin_manual/pages/configuration/user/oidc/oidc.adoc:107]:
```suggestion
NOTE: The desktop and mobile apps (clients) have a default client ID and secret hardcoded, used for ownCloud's oauth2 app. When using Kopano as IDP, it does not predefine a client ID and secret. You can use the default ones from the client to configure Kopano properly. For some IDP's like MS-Azure, these and other required parameters come from the IDP and must be coded into the client. Note that each IDP has different requirements. Get in touch with ownCloud for a branding subscription to customize the clients according your needs.
```

File: modules/admin_manual/pages/configuration/user/oidc/oidc.adoc [hunk 2]
```diff
@@ -104,6 +104,8 @@ xref:configuration/user/oidc/ms-azure-setup.adoc[Example Setup Using Microsoft A
 
 ownCloud desktop and mobile clients detect whether OIDC is available (service discovery) and use this login method when a new account is created.
 
+NOTE: The desktop and mobile apps (clients) have a default client ID and secret hardcoded, used for ownCloud's oauth2 app. When using Kopano as IDP, it does not predefine a client ID and secret. You can use the default ones from the client to configure Kopano properly. For some IDP's like MS-Azure, these and other required parameters come from the IDP and must be coded into the client. Note that each IDP has different requirements. Get in touch with ownCloud for a branding subscription to customize the clients according your needs.
```
Comment (EParzefall) [inline modules/admin_manual/pages/configuration/user/oidc/oidc.adoc:107]:
```suggestion
NOTE: The desktop and mobile apps (clients) have a default client ID and secret hard-coded, which are used for ownCloud's oauth2 app. When using Kopano as IDP, it does not pre-define a client ID and secret. You can use the default ones of the client to configure Kopano properly. With some IDPs like MS-Azure, these and other required parameters come from the IDP and must be coded into the client. Note that each IDP has different requirements. Get in touch with ownCloud for a branding subscription to customize the clients according to your needs.
```

**Review by EParzefall — APPROVED** [review]
Minor language issues.

**Review by phil-davis — APPROVED** [review]
This reads OK to me. Someone else knows if it is exactly correct!

**Pull request merged**